test/extensions/filters/http/common/fuzz/uber_filter.cc

```diff
@@ -91,6 +93,11 @@ void UberFilterFuzzer::fuzz(
 
   // Data path should not throw exceptions.
   if (decoder_filter_ != nullptr) {
+    Http::TestRequestHeaderMapImpl headers;
+    for (const auto& header : downstream_data.headers().headers()) {
+      headers.addCopy(header.key(), header.value());
+    }
```

Comment:
Prefer using `Fuzz::fromHeaders<Http::TestRequestHeaderMapImpl>()`.
Also, why isn't this invoked done by https://github.com/envoyproxy/envoy/blob/24254316bab17cb48d48f88297e91a3dee739f58/test/extensions/filters/http/common/fuzz/http_filter_fuzzer.h#L129 ?

Reply:
`decoder_callbacks_` is available here and not in `HttpFilterFuzzer`-- we could move it, but this does work.